pkg/config/app_connection.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/dapr/dapr/pkg/apphealth"
+)
+
+// Protocol is a communications protocol.
+type Protocol string
+
+const (
```

Comment:
We now have these consts defined [here](https://github.com/dapr/dapr/pull/6512/files#diff-c50c636313dcc3b3caf4afd74464bc3e2ed1ccfbbdb9584bd11aa7f1175665a1R40). Please use these ones (and refactor their location if it makes sense) as we want to avoid duplication here.

Reply:
Actually I would prefer if we refactored `pkg/injector/sidecar/consts.go` to move the consts to `pkg/config` (or even better a sub-package `pkg/config/protocol`).

This way, the runtime pkg doesn't need to import the injector.

Reply:
Makes sense. I refactored the sidecar injector and removed duplication.